database/migrations/2023_03_24_113553_create_ab__articles_table.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('ab_article', function (Blueprint $table) {
+            $table->unsignedBigInteger('id')->primary();
+            $table->string('ab_name', 80)->nullable(false);
+            $table->integer('ab_price')->nullable(false);
+            $table->string('ab_description', 1000)->nullable(false);
+            $table->unsignedBigInteger('ab_creator_id')->nullable(false);
+            $table->foreign('ab_creator_id')->references('id')->on('ab_user');
```

Comment:
Would be better with ->onDelete('cascade'). Because if a user delete his / her account, there will be no article to sell.